docs/src/content/en/docs/frameworks/agentic-uis/copilotkit.mdx

```diff
@@ -11,139 +11,139 @@ import Image from "next/image";
 CopilotKit provides React components to quickly integrate customizable AI copilots into your application.
 Combined with Mastra, you can build sophisticated AI apps featuring bidirectional state synchronization and interactive UIs.
 
-## Create a Mastra Project
-{/*
-LLM CONTEXT:
-This Tabs component shows commands for creating a new Mastra project using different package managers.
-Each tab displays the command for that specific package manager to create a Mastra project.
-This is the first step in setting up Mastra with CopilotKit for building AI copilot applications.
-All commands create the same Mastra project but use different package manager syntax.
-*/}
+## React Frontend Setup
 
-<Tabs items={["npx", "npm", "yarn", "pnpm"]}>
-  <Tabs.Tab>
-    ```bash copy
-    npx create-mastra@latest
-    ```
-  </Tabs.Tab>
+Set up a CopilotKit component in your React frontend
+
+<Tabs items={["npm", "yarn", "pnpm"]}>
   <Tabs.Tab>
-    ```bash copy
-    npm create mastra
+    ```bash copy filename="terminal"
+    npm install @copilotkit/react-core @copilotkit/react-ui
     ```
   </Tabs.Tab>
   <Tabs.Tab>
-    ```bash copy
-    yarn create mastra
+    ```bash copy filename="terminal"
+    yarn add @copilotkit/react-core @copilotkit/react-ui
     ```
   </Tabs.Tab>
   <Tabs.Tab>
-    ```bash copy
-    pnpm create mastra
+    ```bash copy filename="terminal"
+    pnpm add @copilotkit/react-core @copilotkit/react-ui
     ```
   </Tabs.Tab>
 </Tabs>
 
+```tsx filename="components/CopilotKitComponent.tsx" showLineNumbers copy
+'use client';
+import { CopilotChat } from "@copilotkit/react-ui";
+import { CopilotKit } from "@copilotkit/react-core";
+import "@copilotkit/react-ui/styles.css";
 
-Select the agent example when scaffolding your project. This will give you a weather agent.
+export function CopilotKitComponent({ runtimeUrl }: { runtimeUrl: string}) {
+  return (
+    <CopilotKit
+      runtimeUrl={runtimeUrl}
+      agent="weatherAgent"
+    >
+      <CopilotChat
+        labels={{
+          title: "Your Assistant",
+          initial: "Hi! 👋 How can I assist you today?",
+        }}
+      />
+    </CopilotKit>
+  );
+}
+```
 
-For detailed setup instructions, see the [installation guide](/docs/getting-started/installation).
+## Mastra Server Integration
```

Comment:
Is this an integration. Create mastra is for creating a new Mastra application, mastra init is for adding Mastra to an existing application. I'm not clear on what's happening here. Are you adding Mastra to the React frontend, or creating a new Mastra Project